fakeredis/_server.py

```diff
@@ -13,12 +14,21 @@
 from collections import defaultdict
 from collections.abc import MutableMapping
 
+import lupa
```

Comment:
Please remove this and import lupa only where it's needed. It's an optional dependency and fakeredis should work without it as long as Lua functionality isn't needed.

Reply:
Removed